### PR TITLE
Created navigation bar

### PR DESCRIPTION
**Pull request by Akash-Varude**
Review my code
Floated menu button to the right
Created collapsed menu options
Redirected to blogs.html when clicked
Added transition to menu options when hovered

### CODE REVIEW

File: Pages/Shared/layout.html [hunk 1]
```diff
@@ -1,22 +1,38 @@
+<!doctype html>
 <html>
 <head>
     <title>Template</title>
-    <script src="../../Libraries/jquery-3.5.1.min.js"></script>
-    <script>
-        function check() {
-            $('#tar').load('error.html');  // here 'test.html' is a page and 'target' id a id of 'test.html'
```
Comment (Rajashree-Pethkar) [inline Pages/Shared/layout.html:7]: tar is not a valid name. Please add valid names. Also, do not just copy paste the code. Remove all unwanted code and comments.

File: Pages/Shared/layout.html [hunk 2]
```diff
@@ -1,22 +1,38 @@
...
-        }
-    </script>
+    <script src="../../Scripts/jquery-3.5.1.min.js"></script>
```
Comment (Rajashree-Pethkar) [inline Pages/Shared/layout.html:6]: Look for bundling javascript files.

File: Pages/Shared/layout.html [hunk 3]
```diff
@@ -1,22 +1,38 @@
...
-<div>
-   Footer
-</div>
+<body>
+
+    <nav class="navbar  nevigation">
+        <a class="navbar-brand" href="#">Rajashree</a>
+        <img class="menu-image" data-toggle="collapse" data-target="#collapsibleNavbar" src="../../Images/menu-white-18dp.svg" width="30" height="30"/>
+
+        <div class="collapse navbar-collapse" id="collapsibleNavbar">
```
Comment (Rajashree-Pethkar) [inline Pages/Shared/layout.html]: Name should be "collapsible-nav". Please follow this everywhere.

File: Pages/Shared/layout.html [hunk 4]
```diff
@@ -1,22 +1,38 @@
...
-    <h1>Header</h1>
-</div>
-<div id="tar">
-</div>
-<div>
-   Footer
-</div>
+<body>
+
+    <nav class="navbar  nevigation">
```
Comment (Rajashree-Pethkar) [inline Pages/Shared/layout.html]: navigation- spelling mistake.

File: Pages/Shared/layout.html [hunk 5]
```diff
@@ -1,22 +1,38 @@
...
-<div>
-    <h1>Header</h1>
-</div>
-<div id="tar">
-</div>
-<div>
-   Footer
-</div>
+<body>
+
```
Comment (Rajashree-Pethkar) [inline Pages/Shared/layout.html:18]: No need of space. Adding spaces will cause the page to take time to load which affects the performance.

**Review by Rajashree-Pethkar — COMMENTED** [review]
Please implement the changes and send for review. Make sure you follow the comments everywhere.

**Comment by Akash-Varude** [comment]
> 
> 
> Please implement the changes and send for review. Make sure you follow the comments everywhere.

**Pull request closed without merging**